lib/store/index.js

```diff
@@ -7,12 +7,25 @@
 import { createStore, applyMiddleware } from 'redux';
 import reducer from './reducer';
 import thunk from 'redux-thunk';
+import createLogger from 'redux-logger';
```

Comment:
Move this import after line 17, so we will import just if we are using the logger